docs/graphs/graph-sdk-example.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Graph SDK Example
+sidebar_position: 100
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+## Example Overview
+
+This example demonstrates the ability of graph databases in analyzing and offering recommendations for items that are commonly purchased together. By establishing connections between customer and item vertices using edge collections, the graph is able to seamlessly and accurately deliver tailored recommendations to customers who have bought a particular item.
+
+![Create a collection](/img/graphs/sdk-graph-image.png)
+
+In the graph that we created customers Dino and Durga bought the same items. The graph is able to recommend items to Dallas based on the items that Dino and Durga bought. This is a simple example of how graph databases can be used to offer recommendations to customers.
+
+## How to define vertices and edges
+
+For this example we will have two vertices - `groceryCustomers` and `groceryItems` and one edge collection - `groceryOrders`. The `groceryOrders` edge collection will be used to establish a connection between a customer and an item that they bought. Vertex collections are document store collections that store the vertices of a graph. Edge collections are graph edge collections that store the edges of a graph, edge collections have the `_from` and `_to` attributes that are used to establish a connection between two vertices.
```

Comment:
When you say "two vertices," do you mean vertices or vertex collections? 
 
Also, you don't need much basic info. This is an example, not a getting started topic. We cover this information elsewhere.